.github/scripts/common.sh

```
@@ -0,0 +1,2358 @@
+        /home/$current_user/workspace/eloqkv/install/bin/eloqkv \
+        --port=$port \
+        --core_number=2 \
+        --enable_wal=false \
+        --enable_data_store=false \
+        --eloq_data_path="/tmp/redis_server_data_$index" \
+        --event_dispatcher_num=1 \
+        --auto_redirect=true \
+        --maxclients=1000000 \
+        --checkpointer_interval=36000 \
+        --ip_port_list=127.0.0.1:6379,127.0.0.1:7379,127.0.0.1:8379 \
+        --txlog_service_list=$log_service_ip_port \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)